utils/index.js

```diff
@@ -7,6 +7,9 @@ export { focus };
 export { keycodes };
 export { decodeEntities };
 
+// Keep all old wp.utils components
+export const WordCounter = wp.utils.WordCounter;
+
```

Comment:
Unrelated comment:
> export { viewPort };

It probably shouldn't be here anymore :)

Reply:
yep :) it's still used in the image block though.